portality/view/account.py

```diff
@@ -250,6 +259,7 @@ def register():
         account = models.Account.make_account(
             username=form.w.data,
             email=form.n.data,
+            roles=[r.strip() for r in form.roles.data.split(',')]
         )
         account.set_password(form.s.data)
         account.save()
```

Comment:
GH not allowing me to comment on line 257 (new) so just leaving it here.

```form = RegisterForm(request.form, csrf_enabled=False, roles='api')```

That's an interesting way of adding a default value to a field - I think I've seen that, but probably years ago. So you just pass [name of field] = value as further keyword arguments and if the form constructor isn't specifically written to handle them, it just prefills the fields with the keyword args? That's quite cool.